packages/webapp/package.json

```diff
@@ -63,6 +63,7 @@
     "immer": "^9.0.12",
     "moment": "^2.29.1",
     "moment-range": "^4.0.2",
+    "nodemailer": "^6.7.5",
```

Comment:
Since this is for Cypress testing, nodemailer should be a devDependency.
Probably okay to just move the line to the other section, but please check results by completely deleting `webapp/node_modules` then doing `pnpm install`.